native/android/src/main/java/com/nozbe/watermelondb/DatabaseDriver.kt

```diff
@@ -31,7 +31,11 @@ class DatabaseDriver(context: Context, dbName: String) {
     }
 
     constructor(context: Context, dbName: String, schema: Schema) : this(context, dbName) {
-        setUpDatabase(schema)
+        try {
+            unsafeResetDatabase(schema)
+        } catch (e: Exception) {
+            e.printStackTrace()
```

Comment:
I think that leaving error handling to the DatabaseBridge is better idea, then you can propagate error to JS and handle it there. Similar to how you've done `setUpWithMigrations`

Reply:
@rozPierog let's leave it for later, ok? there are more unlikely failures (like failure to open database) that currently will just fail/crash/whatever, and don't propagate to JS… so it's a different feature than this